hoodie-client/src/main/java/com/uber/hoodie/index/bloom/HoodieBloomIndex.java

```diff
@@ -253,8 +253,15 @@ private int determineParallelism(int inputParallelism, int totalSubPartitions) {
         }).collect();
 
     if (config.getBloomIndexPruneByRanges()) {
+      int numSlices = Math.max(dataFilesList.size(), 1);
```

Comment:
Change the name of this variable to rangeBloomIndexParallelism or smthng similar ?